content/articles/changing-domain-contact.md

```diff
@@ -14,86 +14,75 @@ categories:
+1. If you have more than one account, select the relevant one.
+1. On the header, click the <label>Contacts</label> tab, then click <label>New Contact</label>.
+    ![screenshot of adding a contact](/files/new-contacts.png)
 
-    ![Change contact preview](/files/change-contact-preview.png)
+1. Fill in the form with the new contact information, and click <label>Add contact</label>.
+1. Go to your domain page.
+1. On the header, click the <label>Domain Names</label> tab, locate the relevant domain, then click the name to access the domain page.
+1. On the domain details page, click the <label>Registration</label> tab on the left side.
+1. On the card displaying the contact information of your domain, click <label>Change Contact</label>.
+    ![screenshot of changing contact](/files/registration-edit-contact.png)
 
-1.  You will see a confirmation of the new contact information for the domain. If you see a message that the contact change results in the domain being [locked from transfers for 60 days](/articles/icann-60-day-lock-registrant-change/), click the <label>Change Contact</label> link to change the contact.
-
-    ![Change contact confirmation](/files/change-contact-confirmation.png)
+1. You will see a list of all your existing contacts. The one currently assigned to the domain is automatically selected. By selecting a contact, you will see the chosen contact information with its extended attributes. Click <label>Next</label>.
+1. You will see a confirmation of the new contact information for the domain. You may see a message that the contact change results in the domain being [locked from transfers for 60 days](/articles/icann-60-day-lock-registrant-change/). Click <label>Change contact</label> to confirm the change.
 
+    ![screenshot of editing a contact](/files/confirm-contact-change.png)
 </div>
 
-
-
 ## Updating a domain contact
 
-If you want to change existing contact information, just update the contact details. We will immediately update the registry data and the WHOIS record with the new information.
-
+You can update the contact details to change existing contact information. DNSimple will immediately update the registry data and the WHOIS record with the new information.
 <div class="section-steps" markdown="1">
 ##### To change the contact information
 
-1.  Log in to DNSimple with your user credentials.
-1.  On the header, click the <label>Contacts</label> tab, locate the contact you want to change, and click <label>Edit</label>.
-
-    ![Edit contact data](/files/edit-existing-contact-1.png)
+1.  On the header, click <label>Contacts</label>, locate the contact you want to change, and click <label>Edit</label>.
+    ![screenshot of editing domain contact](/files/registration-edit-contact.png)
```

Comment:
@dperrymorrow this is the wrong screenshot, should be https://drive.google.com/file/d/1NVmoae4G9bXBC4kVQtg8uZqxqq08EXdr/view?usp=sharing